src/main/java/com/mraof/minestuck/item/weapon/WeaponItem.java

```diff
@@ -42,14 +48,17 @@
 	@Deprecated
 	public WeaponItem(IItemTier tier, int attackDamage, float attackSpeed, float efficiency, @Nullable MSToolType toolType, Properties properties)
 	{
-		this(new Builder(tier, attackDamage, attackSpeed).efficiency(efficiency).set(toolType), properties);
+		this(new Builder(tier, attackDamage, attackSpeed).efficiency(efficiency).disableShield().set(toolType), properties);
```

Comment:
Adding disable shield in this constructor does not seem right. However this constructor is not meant to be used anymore, so instead of removing just `disableShield()` here, you might as well remove the entire constructor.